config/localization/bundles/src/bundle/org/sakaiproject/localization/bundle/tool/tools.properties

```diff
@@ -249,3 +249,5 @@ sakai.delegatedaccess.shopping.description=Search for available sites for shoppi
 sakai.adminsiteperms.title=Admin Site Perms
 sakai.adminsiteperms.description=Allows adminstrators to control permissions for all sites of a type and roles within that site
 
+sakai.gradebookng = Gradebook
```

Comment:
This key will not work, will it? AFAIK every tool needs to have a ".title" and ".description" endings in their keys.

We have changed it in our local 11.4 to have GradebookNG correctly internationalized.